diff --git a/transport/recovery.go b/transport/recovery.go
index cd3e42e..c12c400 100644
--- a/transport/recovery.go
+++ b/transport/recovery.go
@@ -25,7 +25,7 @@ const (
 	// When no previous RTT is available, the initial RTT should be set to 333ms,
 	// resulting in a 1 second initial timeout
 	// https://www.rfc-editor.org/rfc/rfc9002.html#section-6.2.2
-	initialRTT = 333 * time.Millisecond
+	initialRTT = 100 * time.Millisecond
 
 	// The period of time for persistent congestion to be established,
 	// specified as a PTO multiplier. The recommended value is 3, which is approximately
